single-user-sobigdata/Dockerfile

```diff
@@ -1,7 +1,101 @@
+#   _____       ____  _       _____        _                    
+#  / ____|     |  _ \(_)     |  __ \      | |         _     _   
+# | (___   ___ | |_) |_  __ _| |  | | __ _| |_ __ _ _| |_ _| |_ 
+#  \___ \ / _ \|  _ <| |/ _` | |  | |/ _` | __/ _` |_   _|_   _|
+#  ____) | (_) | |_) | | (_| | |__| | (_| | || (_| | |_|   |_|  
+# |_____/ \___/|____/|_|\__, |_____/ \__,_|\__\__,_|            
+#                        __/ |                                  
+#                       |___/      
+
 ARG BASE_IMAGE=eginotebooks/base:latest
 # hadolint ignore=DL3006
 FROM $BASE_IMAGE
 
+USER root
+
+RUN apt-get update \
+    && apt-get install --no-install-recommends -y \
+        nco \
+        cdo \
+        gdal-bin \
+        cmake \
+        libgdal-dev=3.0.4+dfsg-1build3 \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*
+
+USER $NB_UID
+
+RUN mamba install -y --quiet \
+        basemap \
+        beakerx \
+        xgboost \
+        lightgbm \
+        catboost \
+        netcdf4 \
+        aiohttp \
+        graph-tool \
+        scikit-learn==0.22.1 \
+        scikit-mobility \
+        dask-jobqueue \
+        fs \
+        rasterio \
+        contextily \
+        nb_conda_kernels \
+        nb_conda \
+        pyophidia \
+    && conda clean -tipsy
+
+RUN conda create --quiet --yes -n java8 \
+    && mamba install -n java8 --yes openjdk=8 \
+    && conda clean -tipsy
+
+ENV CPLUS_INCLUDE_PATH=/usr/include/gdal
+ENV C_INCLUDE_PATH=/usr/include/gdal
+
+# Pinned networkx to 2.6.3 as 2.7 makes networksns fail
+RUN pip install --no-cache-dir \
+        bicm \
+        rdp \
+        oceans \
```

Comment:
Oceans is from BlueCloud only

Reply:
OK, I will do the changes... I left everything that was in the Dafault + Training, so this is a good clean up

Reply:
Addressed in commit 33ffcf1b6004b7c629f97fad2db779dafde648b4